code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -654,6 +688,7 @@
 /datum/tarot/reversed/the_high_priestess
 	name = "II - The High Priestess?"
 	desc = "Run."
+	extended_desc = "summons Bubblegum to tear portals open, that will chase the user, and grab people at random, causing major damage."
```

Comment:
This description is unclear to me, Bubblegum will open portals and the portals will chase the user? In the code, it looks like there will be portals opening around the user that attempt to slap anyone nearby, but I cannot see Bubblegum physically appearing.

Reply:
cleared the wording up a tad